files/system/river/usr/bin/startriver

```diff
@@ -5,11 +5,12 @@
 # builds actually ran successfully without any errors!
 set -oue pipefail
 
-
+wayblue_origin_conf="/usr/etc/river/init"
 river_conf="$HOME/.config/river/init"
```

Comment:
Use `${XDG_CONFIG_HOME:-$HOME/.config}` instead of `$HOME/.config` directly. Ik it's not part of the change set but we should fix this while we're looking at it.